examples/vadd/vaddhost.f90

```diff
@@ -0,0 +1,95 @@
+
+#ifdef JIT
```

Comment:
Please comment these various options. We could read the filename from the command line but I think that would be more effort than is necessary.

Reply:
I added a comment explaining the defines. I didn't want to add a command line or environment var to make it easy to execute the example with the standard `make; ./exec;` and not having to remember specific details.

Reply:
Sure, that's fine.